JavaScript/tests/syntax_test_js.js

```diff
@@ -702,6 +702,10 @@ switch ($foo) {
     default$
 //  ^^^^^^^^ - keyword
     ;
+
+    case 0: {}
+    case 1:
+//  keyword.control.switch
```

Comment:
this appears to just be a comment, not a syntax test assertion, it is missing the `^` chars